spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteTablePathsAction.java

```diff
@@ -111,23 +125,20 @@ protected ActionsProvider actions() {
   @BeforeEach
   public void setupTableLocation() {
     this.tableLocation = tableDir.toFile().toURI().toString();
-    this.table = createATableWith2Snapshots(tableLocation);
```

Comment:
I feel like we should set up the table here instead of in every single test method

Reply:
+1 If we extend the "bottom" `createTableWithSnapshots` at line 149 with the `format-version` property, then it will be valid for all the tests within this suite.